app/views/rubygems/show.html.erb

```diff
@@ -147,16 +147,13 @@
         <%= link_to t('edit'), edit_rubygem_path(@rubygem), :class => "gem__link t-list__item", :id => "edit" if @rubygem.owned_by?(current_user) %>
 
         <% if @latest_version.indexed %>
-          <% if @rubygem.linkset.present? %>
-            <%- %i{home code docs wiki mail bugs}.each do |link| %>
-              <%= link_to_page t("rubygems.show.links.#{link}"), @rubygem.linkset.public_send(link) %>
-            <%- end %>
-          <% end %>
+          <%- Linkset::LINKS.each do |link, attribute| %>
```

Comment:
Big fan of this, literally just suggested this in the original commit today. :+1: